### PR TITLE
nova: Add option to disable ephemeral on ceph (SOC-5269)

### PR DESCRIPTION
**Pull request by skazi0**
New attribute was added to enable/disable ephemeral storage on CEPH/SES if
needed.

Added UI part:
Zrzut ekranu z 2020-03-17 23-08-25

bonus: remove redundant config line

### CODE REVIEW

File: chef/data_bags/crowbar/migrate/nova/306_add_use_rbd_ephemeral.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def upgrade(template_attributes, template_deployment, attributes, deployment)
+  key = "use_rbd_ephemeral"
+  attributes[key] = template_attributes[key] unless attributes.key? key
```
Comment (JanZerebecki) [inline chef/data_bags/crowbar/migrate/nova/306_add_use_rbd_ephemeral.rb]:
```suggestion
  attributes[key] = false unless attributes.key? key
```
Then set the template to true, which would have the same effect as setting it in the model to true only on new installs. Not sure right now which one is better.

Comment (skazi0) [reply chef/data_bags/crowbar/migrate/nova/306_add_use_rbd_ephemeral.rb]: Common pattern is to have migrations use default from template. I added enabling this for new deployments in `create_proposal()`

**Comment by skazi0** [comment]
I've added a custom view control for the new flag. Please re-review.

**Comment by skazi0** [comment]
backport added to https://github.com/crowbar/crowbar-openstack/pull/2364

**Pull request merged**